doc/releases/changelog-0.23.0.md

```diff
@@ -420,6 +403,40 @@
    expval(PauliZ(wires=[0]))]
   ```
 
+* Added the `QuantumTape.shape` method and `QuantumTape.numeric_type`
+  attribute to allow extracting information about the shape and numeric type of
+  quantum tapes.
+  [(#2044)](https://github.com/PennyLaneAI/pennylane/pull/2044)
+
+* Defined a `MeasurementProcess.shape` method and a
+  `MeasurementProcess.numeric_type` attribute.
+  [(#2044)](https://github.com/PennyLaneAI/pennylane/pull/2044)
```

Comment:
This should specify _what_ these attributes allow you to extract! Like in the above docstring.